src/components/AccountForms/RegistrationForm/index.js

```diff
@@ -91,11 +91,11 @@ const useStyles = makeStyles((theme) => ({
     color: '#550CCC',
   },
   button: {
-    [theme.breakpoints.up(799)]: {
-      marginTop: '48px',
-      float: 'right',
+    [theme.breakpoints.up(824)]: {
       marginRight: '16.25em',
     },
+    marginTop: '48px',
+    float: 'right',
```

Comment:
we should generally avoid float as it causes hard to debug style issues down the line

we could achieve the same thing in a more controlled manner with
```
display: flex;
justify-content: flex-end;
```

Reply:
resolved in commit # https://github.com/cfernando415/femmecubator-ui-repo/commit/effe3e9fe18a2a03c2c5ef8ff1cceaabb6484422